addon-test-support/wait-until.js

```diff
@@ -5,7 +5,7 @@ export function waitUntil(callback, { timeout = 1000 } = {}) {
   return new RSVP.Promise(function(resolve, reject) {
     let value = run(callback);
     if (value) {
-      resolve(value);
+      return resolve(value);
```

Comment:
Exactly. The code was technically fine, but it might have been confusing for future readers (possibly including myself 😝).